components/stepper/stories/template.js

```diff
@@ -74,6 +74,7 @@ export const Template = ({
 				value: "0",
 				isDisabled,
 				isQuiet,
+				...(id ? {id:`${id}-input`} : {}), 
```

Comment:
```suggestion
				id: id ? `${id}-input` : undefined,
```

This outputs basically the same thing but it might be just a little easier to read for someone trying to debug the file later.

Reply:
Good suggestion. I've made this change.